src/repositories/models.py

```diff
@@ -1,17 +1,31 @@
 from mongoengine import *
+from enum import Enum
 
-class Organization(Document):
+
```

Comment:
Both `Orgnization` and `Machine` objects should be embedded object under the `Sensor` object